### DISCUSSION TITLE
Distance units for Parsec and Astronomical unit

### DISCUSSION THREAD

**Discussion by jschlyter** [home-assistant/architecture#1166] [Entity Models]
I suggsst we add parsec (pc) and astronomical unit (au) as distance units. These are both non-US and non-metric units, and added for completeness and because they are interesting and fun. They are probably not applicable to most home automation installations unless you're running Home Assistant on the [International Space Station](https://www.nasa.gov/international-space-station/) or measuring distance to other galaxies.

**Not adding these units will most likely have zero impact**, but on the other hand adding them does not break anything (and will teach us to use very big and small numbers).

Closed PR in https://github.com/home-assistant/core/pull/129187 if someone wants to take a look.

**Comment by AJediIAm** [comment]
I did a bit of searching and there are a few HACS integrations which could make use of this.

https://github.com/partofthething/ha_skyfield?tab=readme-ov-file
And
https://github.com/mawinkler/astroweather
Show the position of planets. 
The Skyfield library https://rhodesmill.org/skyfield/ includes distance in au. By adding the units, the listed integrations could show the distance to the planets.
